src/Admin/AbstractAdmin.php

```diff
@@ -709,7 +709,7 @@ public function getFilterParameters()
                 }
             }
 
-            $parameters = array_merge($parameters, $filters);
+            $parameters = $this->mergeParameters($parameters, $filters);
```

Comment:
Should we do a static methd in a service instead.
Like TraversableToCollection::transform.

AbstractAdmin is already a huge class.

Reply:
Good idea! Moved the method to a separate class ParametersManipulator.